master/templates/kafka/kafka.yml

```diff
@@ -0,0 +1,72 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: "{{ .Values.kafka.strimziCluster }}"
+spec:
+  kafka:
+    replicas: {{ .Values.kafka.defaultReplicas }}
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      num.partitions: 30
+      default.replication.factor: 3
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+      auto.create.topics.enable: false
+    storage:
+      type: persistent-claim
+      size: {{ .Values.kafka.storageSize }}
+      deleteClaim: false
+    readinessProbe:
+      initialDelaySeconds: 30
+      timeoutSeconds: 15
+      failureThreshold: 20
+    livenessProbe:
+      initialDelaySeconds: 30
+      timeoutSeconds: 15
+    resources:
+{{ toYaml .Values.kafka.resources | indent 6 }}
+    # rack:
+    #   topologyKey: failure-domain.beta.kubernetes.io/zone
+    tlsSidecar:
+{{ toYaml .Values.tlsSidecar | indent 6 }}
+#     tolerations:
+# {{ toYaml .Values.tolerations | indent 6 }}
+#     affinity:
+# {{ toYaml .Values.affinity | indent 6 }}
```

Comment:
What are we doing with these lines commented out? We should probably remove them.

Reply:
From memory, this was because we thought the kafka should be properly open sourced/overridden and we'd need these values in that case so we didn't want to remove them (since the squash would remove them from git history). I can't fully remember the reasoning though so I'm game to delete them.

Reply:
I deleted all the comments